cf_units/__init__.py

```diff
@@ -1849,10 +1849,10 @@ def __ne__(self, other):
         """
         return not self == other
 
-    def convert(self, value, other, ctype=FLOAT64):
+    def convert(self, value, other, ctype=FLOAT64, inplace=False):
         """
-        Converts a single value or numpy array of values from the current unit
-        to the other target unit.
+        Converts a single ``value`` or NumPy array of ``value``s from the
```

Comment:
I thought we used double back-ticks to denote actualt variables or values like ``False`` or ``None``, I'm not sure they are appropriate here as the word value is generic and not referring to the input argument ``value``.